cms/templates/js/signatory-details.underscore

```diff
@@ -15,7 +15,7 @@
         </div>
         <div>
           <span class="signatory-title-label"><b><%= gettext("Title") %>:</b>&nbsp;</span>
-          <span class="signatory-title-value"><%= title %></span>
+          <span class="signatory-title-value"><%= title.replace(new RegExp('\r?\n','g'), '<br />') %></span>
         </div>
```

Comment:
it replaces new line characters (i.e. \n) with html line break tag, I will also add a comment here.